src/components/Inputs/inputs.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+const monthList = [
+  {id: "1", month:"January"},
+  {id: "2", month:"February"},
+  {id: "3", month:"March"},
+  {id: "4", month:"April"},
+  {id: "5", month:"May"},
+  {id: "6", month:"June"},
+  {id: "7", month:"July"},
+  {id: "8", month:"August"},
+  {id: "9", month:"September"},
+  {id: "10", month:"October"},
+  {id: "11", month:"November"},
+  {id: "12", month:"December"}
+];
```

Comment:
I see 2 options here. 
We can move this to its own dedicated `constants` file where we could import this or use a library like [MomentJS](https://momentjs.com/) to acquire this kind of data (It's also pretty useful to parse dates and make transformations for them)

What do you think, @luisalduucin, @mirantrix  ?

Reply:
ok I will take a look at the library, thnx!